python/feathub/processors/local/tests/test_local_processor_over_window.py

```diff
@@ -0,0 +1,832 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from datetime import timedelta
+from math import sqrt
+
+import pandas as pd
+
+from feathub.common.test_utils import LocalProcessorTestCase
+from feathub.common.types import Int64, String, Float64, MapType
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.transforms.over_window_transform import OverWindowTransform
+from feathub.feature_views.transforms.python_udf_transform import PythonUdfTransform
+from feathub.online_stores.memory_online_store import MemoryOnlineStore
+
+
+class FlinkTableBuilderOverWindowTransformTest(LocalProcessorTestCase):
```

Comment:
Update the class name as appropriate?

Can you also update the file name of this and existing files so that it is clear what is the relationship/difference between this test and `test_local_processor.py`?